pkg/meta/redis.go

```diff
@@ -3027,6 +3076,7 @@ func (m *redisMeta) LoadMeta(buf []byte) error {
 		return err
 	}
 
+	dm.FSTree.Attr.Inode = 1
```

Comment:
maybe also reset mode, uid & gid? i.e. mode=0777, uid=gid=0

Reply:
The attribute of root could be change to anything, so we can't reset it.